README.md

```diff
@@ -161,7 +165,7 @@ kubectl apply -f example-resources/namespaces/another-namespace.yaml
 
 Rollout Nginx. This will produce a commit in `example-resources/`:
 ```
-./ausroller.py --config example-resources/ausroller.ini --namespace another-namespace --app nginx --version 1.10.2-alpine
+ausroller --config example-resources/ausroller.ini --namespace another-namespace --context another-context --app nginx --version 1.10.2-alpine
```

Comment:
Please you `minikube` instead of `another-context` to be aligned with the example `ausroller.ini`.